Sources/Views/MessageLabel.swift

```diff
@@ -322,6 +353,36 @@ open class MessageLabel: UILabel {
         return results
     }
 
+    /**
+     Simply check if the detector type is a .custom
+     All .custom contains an `NSRegularExpression`
+
+     - Parameters: detector: `DetectorType` that you want to execute
+
+     - Returns: Bool true if the detector type is a .custom, otherwise false
+     */
+    private func isCustomDetector(for detector: DetectorType) -> Bool {
```

Comment:
@nathantannar4 may be we can do this operation in one line of code

Reply:
I think this should be a property of `DetectorType`, not a method in `MessageLabel`

```
enum DetectorType {
...

     var isCustom: Bool {
                   ...
     }
```